src/main/java/com/auth0/json/mgmt/jobs/UsersExportField.java

```diff
@@ -0,0 +1,39 @@
+package com.auth0.json.mgmt.jobs;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Class that represents a Users Export Job's Field object. Related to the {@link com.auth0.client.mgmt.JobsEntity} entity.
+ */
+@SuppressWarnings({"unused", "WeakerAccess"})
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class UsersExportField {
+
+    @JsonProperty("name")
+    private String name;
+    @JsonProperty("export_as")
+    private String exportAs;
+
+    public UsersExportField(String name) {
```

Comment:
Keeping this one handy as seems to be the default scenario. If the alternative constructor is not required, I can replace it with a setter for `export_as`